.pylint-ignore.dict

```diff
@@ -0,0 +1,21 @@
+param
+sa
+numpy
+py
+args
+kwargs
+ui
+bool
+reimplemented
+gil
+osx
+traceback
+shaders
+whitespace
+gettext
+i18n
+catalog
```

Comment:
Just "english". Could be that it is simply a bug in the aspell english dictionary.